target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -217,6 +216,11 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "4:wan" "6@eth0"
 		;;
+	youku,yk-l1|\
+	youku,yk-l1c)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "6@eth0"
```

Comment:
If you intend to change the network config, please do that in a separate (proper) commit.

Reply:
okay, I will obey the rules. I'll change the commit soon. But before that I really need to learn more git operate tips😂